packages/shared/src/utils/url/__tests__/url.js

```diff
@@ -48,7 +38,7 @@ describe('Url', () => {
             });
             describe('.urlFor()', () => {
                 it('returns home as default', () => {
-                    [undefined, null, '', '/', 'home'].forEach(path => {
```

Comment:
We shouldn't pass null | undefined here!

Reply:
@niloo-fs it's simply used to check these: `expect(urlFor(undefined)).to.eq(home_url);` and `expect(urlFor(null)).to.eq(home_url);`
It's good to check against undefined and null, ain't it? 😊

Reply:
So if I want to keep these 2 I need to change the type of path from string to string | null | undefined! and change the urlFor method to handle undefined and null paths

Reply:
Anytime